web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -640,6 +692,9 @@ const TeamReviews = ({ onBack, periodId }) => {
             label={reviewer.name}
             variant="outlined"
             onDelete={() => deleteReviewer(member, reviewer)}
+            style={{
```

Comment:
This sets the background color of each reviewer `Chip` based on whether the assignment has been approved.

Reply:
When the background color is changed and user prefers dark mode, to the light green/yellow colors here still contrast as expected with the test color (which at this point I would suspect is white/off-white)?

Reply:
Great catch! This has been fixed in the latest commit. There is still an issue with DatePicker components in dark mode. I created a new issue for that and Jack will look into it. https://github.com/objectcomputing/check-ins/issues/2386